web/src/message_edit.ts

```diff
@@ -668,6 +668,10 @@ function edit_message($row: JQuery, raw_content: string): void {
         $message_edit_content.on("keyup", (event) => {
             compose_ui.handle_keyup(event, $message_edit_content);
         });
+        compose_tooltips.initialize_compose_tooltips(
+            "edit_message",
+            ".message_edit .compose_button_tooltip",
+        );
```

Comment:
This doesn't work because you can have multiple edit message widgets open. Look at how `compose_validate` or `upload.ts` handle this issue and you'll see you can fix by including the message ID in your `SingletonContext` type key.